apps/openmw/mwrender/characterpreview.cpp

```diff
@@ -129,7 +129,7 @@ namespace MWRender
                     }
                     // Disable noBlendAlphaEnv
                     newStateSet->setTextureMode(7, GL_TEXTURE_2D, osg::StateAttribute::OFF);
-                    newStateSet->addUniform(mNoAlphaUniform);
+                    newStateSet->setDefine("TRANSLUCENT_FRAMEBUFFER", "0", osg::StateAttribute::ON);
```

Comment:
With the define and uniform separate, they described two separate things - the define covered whether the framebuffer had an alpha channel, and the uniform whether the current object should only emit opaque fragments. If both concerns are covered by one define, the name could be a little clearer. It's not like the framebuffer suddenly stops being translucent when blending gets turned on.